crash-reporting-core/src/androidMain/kotlin/dev/icerock/moko/crashreporting/core/getStackTrace.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.crashreporting.core
+
+actual fun CrashReportingCore.getStackTrace(throwable: Throwable): List<Long> {
+    // Android don't need stack trace
+    return listOf()
```

Comment:
if on androit it's not used - we should not declare it as `expect` in common code and do empty `actual` - just declare only `iosMain` fun